src/System.IO.Compression/ref/4.0.0/System.IO.Compression.depproj

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <OutputType>Library</OutputType>
+    <PackageTargetFramework>dotnet5.2</PackageTargetFramework>
```

Comment:
> 4.0.0 that first shipped in .NET 4.5 which is gen 5.2

Look, everything that's wrong with .NET version numbering in one sentence :laughing: 

Joking aside, is there an ETA on when you'll start documenting how all of this works? It's creating major confusion in the community.